ci/matlab.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow uses actions that are not certified by GitHub.  They are
+# provided by a third-party and are governed by separate terms of service,
+# privacy policy, and support documentation.
+#
+# This is a basic workflow to help you get started with MATLAB actions
+# For more information see: https://github.com/matlab-actions/overview
+
+name: MATLAB
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # Set up MATLAB on the GitHub hosted runner
```

Comment:
1) I think "the" should be replaced with "a".

2) "GitHub hosted" should be hyphenated: GitHub-hosted

Reply:
Sure, thanks!